src/datafactory/azext_datafactory/tests/latest/preparers.py

```diff
@@ -0,0 +1,159 @@
+# --------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+#
+# Code generated by Microsoft (R) AutoRest Code Generator.
+# Changes may cause incorrect behavior and will be lost if the code is
+# regenerated.
+# --------------------------------------------------------------------------
+
+import os
+from datetime import datetime
+from azure_devtools.scenario_tests import SingleValueReplacer
+from azure.cli.testsdk.preparers import NoTrafficRecordingPreparer
+from azure.cli.testsdk.exceptions import CliTestError
+from azure.cli.testsdk.reverse_dependency import get_dummy_cli
+
+
+KEY_RESOURCE_GROUP = 'rg'
+KEY_VIRTUAL_NETWORK = 'vnet'
+KEY_VNET_SUBNET = 'subnet'
+KEY_VNET_NIC = 'nic'
+
+
+class VirtualNetworkPreparer(NoTrafficRecordingPreparer, SingleValueReplacer):
```

Comment:
remove this file if not used.

Reply:
@changlong-liu Will the current preparer generation logic generate all type of Preparer no matter it's used or not?

Reply:
removed.